homeassistant/components/overkiz/quality_scale.yaml

```diff
@@ -0,0 +1,68 @@
+rules:
+  # Bronze
+  config-flow: done
+  brands: done
+  dependency-transparency: done
+  common-modules: done
+  has-entity-name: done
+  action-setup:
+    status: exempt
+    comment: The integration does not register actions.
+  appropriate-polling: done
+  test-before-configure: done
+  entity-event-setup: done
+  unique-config-entry: done
+  entity-unique-id: done
+  docs-installation-instructions: done
+  docs-removal-instructions: done
+  test-before-setup: done
+  docs-high-level-description: done
+  config-flow-test-coverage: done
+  docs-actions:
+    status: exempt
+    comment: The integration does not register actions.
+  runtime-data: done
+
+  # Silver
+  log-when-unavailable: done
+  config-entry-unloading: done
+  reauthentication-flow: done
+  action-exceptions:
+    status: exempt
+    comment: The integration does not register actions.
+  docs-installation-parameters: done
+  integration-owner: done
+  parallel-updates: todo
+  test-coverage: todo
+  docs-configuration-parameters:
+    status: exempt
+    comment: The integration does not provide configuration parameters.
+  entity-unavailable: done
+
+  # Gold
+  docs-examples: todo
+  discovery-update-info: done
+  entity-device-class: done
+  entity-translations: todo
+  docs-data-update: done
+  entity-disabled-by-default: done
+  discovery: done
+  exception-translations: todo
+  devices: done
```

Comment:
Should scenes be attached to a device?

Reply:
No, scenes created in Overkiz can be linked to multiple devices and we can't identify to which devices they are linked. The API only provides the name of the scene.